commands/Tools/contact.js

```diff
@@ -0,0 +1,34 @@
+const types = {
+  bug: "Bug Report",
+  idea: "New Idea",
+  message: "New Message",
+};
+exports.run = async (client, msg, [type, ...content]) => {
+  this.channel.send(`${types[type]} From: ${msg.author} in ${msg.guild ? msg.guild : "private message"}\n\n ${content.toString().join(" ")}`);
```

Comment:
await this, a failure will make it throw an uncaught error.
Also, `content.toString()` will join the words by `,`, and String.prototype.join() is not a method as far as I am aware of.